app.py

```diff
@@ -0,0 +1,28 @@
+from flask import Flask, render_template, jsonify
+import diff_logic as diff_logic
+import json
+
+app = Flask(__name__)
+
+dir_old = 'test/test_old'
+dir_new = 'test/test_new'
+
+
+diff_result = diff_logic.create_folder_diff(dir_old, dir_new)
```

Comment:
Run this in `index` and `example` (Probably should give those routes a better name).  Then find way (eg. input tags) to set any folder path. Then to specify a git folder and a specific branch. Perhaps dulwich can help maybe.